mir_eval/util.py

```diff
@@ -621,9 +621,120 @@ def match_events(ref, est, window):
     # Construct the graph input
     G = {}
     for ref_i, est_i in zip(*hits):
-        if ref_i not in G:
-            G[ref_i] = []
-        G[ref_i].append(est_i)
+        if est_i not in G:
+            G[est_i] = []
+        G[est_i].append(ref_i)
+
+    # Compute the maximum matching
+    matching = sorted(_bipartite_match(G).items())
+
+    return matching
+
+
+def match_notes(ref_intervals, ref_pitches, est_intervals, est_pitches,
+                onset_tolerance=0.05, pitch_tolerance=50.0, offset_ratio=0.2,
+                with_offset=False):
```

Comment:
See my previous comment about this, I prefer to keep `with_offset` unless there's serious opposition.